### ISSUE TITLE
Wikis

### ISSUE THREAD

**Issue by HuseyinSimsek7904** [Duck-Tech-Dev/app-winnerduck#14]
We should create a wiki directory inside this repository just like we did with the api-winnerduck. This wiki should contain `starting-development.md` so that every newcomer can start developing quickly.

**Comment by ErtyumPX** [comment]
You are right, it could be better to write brief docs about the stuff we are using and the conventions etc.

Nevertheless, I'm making this issue `p2` since I'm already handling most of the stuff about front-end and trying to write as much detail as possible for PR's to be reviewed easily, backend part needs the most of the attention for now.

**Comment by HuseyinSimsek7904** [comment]
Thank you.

I also noticed that this repository also uses _.env_ file. Please write a wiki for that as well.
